### DISCUSSION TITLE
How to do orm caching in axum?

### DISCUSSION THREAD

**Discussion by lidh15** [tokio-rs/axum#1689] [Q&A] [answered]
We have some urls provide with sql queries, which basically execute sql and return things with SeaORM or other ORM utilities.
These APIs are frequently requested and in Java we can have a redis backend cache and put some ORM results in redis to accelerate the reponses, is there any traits help to do so in axum?

**Comment by davidpdrsn** [comment, marked as answer]
axum has no opinions on how to cache things. So you can use redis as well.

**Reply by lidh15** [reply]
Okay that's fine, I guess redis-rs is what I should learn, am I right?

**Reply by davidpdrsn** [reply]
🤷

**Reply by davidpdrsn** [reply]
Probably. That one seems to have lots of users. I've also used https://crates.io/crates/redis-async which worked well. Its really up to you.
